src/propagate_constant.cpp

```diff
@@ -36,26 +36,37 @@ inline namespace MIGRAPHX_INLINE_NS {
 
 MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_TRACE_PROPAGATE_CONSTANT)
 
+static int const_pr_ct = 0;
```

Comment:
Dont use a global variable, this is not thread-safe, and this pass is called across multiple threads.